.github/workflows/build.yml

```diff
@@ -53,3 +53,20 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           delete-old-comments: true
           update-comment: true
+  changelog:
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Code Coverage Report Updater
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Changelog check
+        uses: Zomzog/changelog-checker@v1.2.0
+        with:
+          fileName: CHANGELOG.md
+          noChangelogLabel: Missing update to CHANGELOG.md
```

Comment:
The naming of this option is confusing
From the docs:
> The check can be disabled for a PR with a label. This label can be customized with the `noChangelogLabel` parameter.
https://github.com/Zomzog/changelog-checker/blob/master/action.yml#L12

So no need for it
I agree they should have added an option to customize the title

Reply:
mmmm but i have seen that going out in the console? let me check again

Reply:
i definetely got confused with this:
<img width="396" alt="image" src="https://user-images.githubusercontent.com/1194048/191232899-f3ec1b50-f0da-4f04-bb50-11947dd2a2d3.png">

i simply copied their example and smashed in, i didn't even search for a doc page

Reply:
initially i tought this would be the message that goes on the github check section, when failed. Then i see it wasn't and moved on